common/utils/src/main/resources/error/error-conditions.json

```
@@ -5010,6 +5010,43 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_SHOW_CREATE_TABLE" : {
+    "message" : [
+      "Unsupported show create table command."
+    ],
+    "subClass" : {
+      "ON_DATA_SOURCE_TABLE_WITH_AS_SERDE" : {
+        "message" : [
+          "<table> is a Spark data source table. Please use `SHOW CREATE TABLE` without `AS SERDE` instead."
```

Please, follow to `toSQLStmt()`, and just upper case the SQL statements:
```suggestion
          "The table <tableName> is a Spark data source table. Please use SHOW CREATE TABLE without AS SERDE instead."
```